dask_ml/cluster/k_means.py

```diff
@@ -23,6 +25,9 @@
 
 logger = logging.getLogger(__name__)
 
+if DASK_VERSION < packaging.version.parse("1.1.0"):
```

Comment:
Small nitpick. Can you move this to _compat.py and just import blockwise from there? That

1.  Doesn’t change the user’s dask.array, if they’re using it directly
2. Makes future cleanup easier since it’s all in one place.

Reply:
So _compat will have a top-level blockwise method, and we use that everywhere.

Reply:
Yeah, can do. Thanks for suggesting this change